test-cypress.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
I would remove all logic around scenarios. It's not needed for this repo. You can probably remove this script entirely and just use test-prow.sh.